docs/dev/checks/python/check_api.md

```diff
@@ -164,6 +164,27 @@ following:
   configuration file (to be backwards compatible we agent5 checks we have to
   pass a list here).
 
+## Running subprocesses
+
+Because of the embedded and multi-threaded nature of the Python runtime in Agent v6 there are some
```

Comment:
I'd maybe change this slightly, for the sake of precision, to:
```
Due to the python interpreter being embedded in an inherently multi-threaded environment (the go-runtime) there are some limitations to the ways in which we can run subprocesses from Checks (as subprocess.Popen is not thread-safe in python 2.7)....  
```
These being dev docs, I'm not sure if we want to add additional details regarding the nature of the thread unsafety in `subprocess.Popen`  (probably not, but I added up there in parentheses)

Reply:
Good comment thanks. I'll leave out the details on `subprocess.Popen`, as you wrote I think this kind of details are not necessary here.